Sources/SwiftDocC/Semantics/Reference/TabNavigator.swift

```diff
@@ -61,6 +61,11 @@ public final class TabNavigator: Semantic, AutomaticDirectiveConvertible, Markup
 extension TabNavigator {
     /// A container directive that holds general markup content describing an individual
     /// tab within a tab-based layout.
+    ///
+    ///  To nest a new tab within the content of a ``TabNavigator``, add  a `@Tab` directive within the content of a `@TabNavigator` directive.
```

Comment:
I feel like using the phrase "the content" for both the presentation of a tab navigator and for the scope of child directives is adding confusion here.

```suggestion
    ///  To add a new tab to a ``TabNavigator``, add  a `@Tab` directive within the content of the `@TabNavigator` directive.
```